core/core-funcs.el

```diff
@@ -119,6 +119,14 @@ Supported properties:
   (re-search-forward anchor-text)
   (beginning-of-line)
   (show-subtree)
+  (setq-local org-emphasis-alist '(("*" bold)
+                                   ("/" italic)
+                                   ("_" underline)
+                                   ("=" org-verbatim verbatim)
+                                   ("~" org-kbd)
+                                   ("+"
+                                    (:strike-through t))))
+
```

Comment:
oh...my eyes rejoice! :D this looks cool thanks @tuhdo

On Tue, Jun 9, 2015 at 5:49 AM, Tu Do <notifications@github.com> wrote:

> In core/core-funcs.el
> <https://github.com/syl20bnr/spacemacs/pull/1906#discussion_r32001148>:
>
> > @@ -119,6 +119,14 @@ Supported properties:
> >    (re-search-forward anchor-text)
> >    (beginning-of-line)
> >    (show-subtree)
> > +  (setq-local org-emphasis-alist '(("*" bold)
> > +                                   ("/" italic)
> > +                                   ("_" underline)
> > +                                   ("=" org-verbatim verbatim)
> > +                                   ("~" org-kbd)
> > +                                   ("+"
> > +                                    (:strike-through t))))
> > +
>
> Of course, since we still haven't converted any markdown file to org, so
> it opens Org. Try open VIMUSERS.org and you will see something different.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/syl20bnr/spacemacs/pull/1906/files#r32001148>.
>